src/main/scala/com/workday/warp/persistence/mysql/HasWarpSlickDsl.scala

```diff
@@ -52,6 +52,44 @@ trait HasWarpSlickDsl {
       expression.apply(timestamp)
     }
 
-  }
 
+    /**
+      *  Translates to YEAR() function
+      *  @return just the year in Int
+      */
+    def getYear (): Rep[Int] = {
```

Comment:
also for simplicity's sake, let's also try to keep the method names identical to the mysql operators as much as possible here. so simply, `getYear` -> `year()`, `getDate` -> `date()`, `currentTimestamp` -> `now()`